src/main/java/htsjdk/variant/variantcontext/VariantContextUtils.java

```diff
@@ -223,6 +224,23 @@ public JexlVCMatchExp(String name, Expression exp) {
         }
     }
 
+    public enum JexlMissingValueTreatment {
```

Comment:
That was my initial thought, but JEXLMap is package private so it's not a good place to expose anything to a public API